### PR TITLE
fix _Opt deprecation warning

### LINKED ISSUE #234: Resolve deprecation warning
```
.../site-packages/notfound/extension.py:269: RemovedInSphinx90Warning: The '_Opt' object tuple interface is deprecated, use attribute access instead for 'default', 'rebuild', and 'valid_types'.
  default, rebuild, types = app.config.values.get('notfound_urls_prefix')
```

Version 1.0.1

### PR DESCRIPTION
**Pull request by drts01**
closes #234

### CODE REVIEW

**Review by humitos — COMMENTED** [review]
Thanks for this work!

Can you take a look at the documentation build error?

```
   File "/home/docs/checkouts/readthedocs.org/user_builds/sphinx-notfound-page/envs/236/lib/python3.10/site-packages/notfound/extension.py", line 274, in validate_configs
    app.config.values.get("notfound_urls_prefix").default
AttributeError: 'tuple' object has no attribute 'default' 
```

https://beta.readthedocs.org/projects/sphinx-notfound-page/builds/24464999/#231390753--9

It seems that in some versions of Sphinx doesn't have `.default` attribute.

**Comment by drts01** [comment]
Yes. the attribute `default` was added in Sphinx 7.2. It seems one of the Sphinx extensions was causing the issue. Once the dependencies were updated, it now works as expected.

**Review by humitos — APPROVED** [review]
Thanks!

File: notfound/extension.py [hunk 1]
```diff
@@ -266,14 +269,22 @@ def validate_configs(app, *args, **kwargs):
...
+        app.config.values.get("notfound_urls_prefix").default
+        if sphinx.version_info >= (7, 2)
+        else app.config.values.get("notfound_urls_prefix")[0]
+    )
+
+    if (
+        notfound_urls_prefix != default
+        and notfound_urls_prefix
+        and not (
+            notfound_urls_prefix.startswith("/") or notfound_urls_prefix.endswith("/")
```
[the remark this answers is not in the document]
Comment (drts01) [reply notfound/extension.py:283]: oh dang. you're right. idk what i was thinking.

**Comment by hugovk** [comment]
Please could you make a release with this fix? Thank you!

**Comment by humitos** [comment]
@hugovk done -- please confirm it does work as expected 👍🏼

**Comment by hugovk** [comment]
No more warning with 1.0.3, thank you!

**Pull request merged**